### ISSUE TITLE
[Usable - Bot Framework Web Chat - Card Breakfast]: There is no character limit for the 'Type your message' edit field under 'White-labelled' and 'Copilot' bots.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5369]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Allowing the users to enter the message without any character limit can result in poor user experience. Users typically expect input fields to have validation rules in place to ensure the data they enter is accurate and in the expected format. Allowing invalid input can lead to confusion, errors, and frustration.
### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated, Suggested action

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify whether the character limit or not.

### What do you expect?

There should be a character limit for the 'Type your message' edit field under 'White-labelled' and 'Copilot' bots.

### What actually happened?

There is no character limit for the 'Type your message' edit field under 'White-labelled' and 'Copilot' bots.
**Observation:** This issue does not reproes with 'Fluent UI'.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/c925db0c-06a4-4134-9568-1312888ee9fa)
![Image](https://github.com/user-attachments/assets/b9863b11-2079-431d-9953-f3e31ea86f88)
![Image](https://github.com/user-attachments/assets/144020d7-f188-4b31-a863-2a0715a2d1e7)

**Comment by KattaBhargav** [comment]
#A11yUsable;#A11ySev4;#HCL;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#PWD#Win11-Edge(Chromium);
